torch/onnx/symbolic_opset9.py

```diff
@@ -633,6 +633,20 @@ def neg(g: jit_utils.GraphContext, self):
 @_onnx_symbolic("aten::sqrt")
 @_beartype.beartype
 def sqrt(g: jit_utils.GraphContext, self):
+    # TODO(justinchuby): Add a helper function in jit_utils and use it to check types
```

Comment:
Consider adding all jit scalar functionality within `JitScalarType` to increase cohesion. `_is_in_type_group` and friends have added quiet noise to the symbolics and refactoring was challenging